src/components/customSearch/CustomSearch.tsx

```diff
@@ -0,0 +1,213 @@
+                    borderColor: theme => theme.palette.primary.main
+                  }}
+                  href={'/search'}
+                >
+                  <Typography variant="button">SIMPLE SEARCH</Typography>
+                </Button>
+              </Grid>
+            </Grid>
+          </Grid>
+          <Divider />
+          <Formik initialValues={initialValues} onSubmit={handleOnFormSubmit}>
+            {({ values, handleChange, setFieldValue }) => (
+              <Form>
+                <Box
+                  sx={{
+                    width: '100%',
+                    borderRadius: '4px',
+                    boxShadow: '0px 0px 0px 1px #E0E0E0',
+                    mt: 4,
+                    padding: 2,
+                    display: 'flex'
+                  }}
+                >
+                  <Grid container direction="column" width="100%">
+                    <Grid
+                      item
+                      container
+                      direction="row"
+                      justifyContent="flex-start"
+                      width="fit-content"
+                    >
+                      <Grid item>
+                        <Typography
+                          variant="h5"
+                          sx={{
+                            color: 'rgba(0, 0, 0, 0.6)'
+                          }}
+                        >
+                          Custom Patient Search
+                        </Typography>
+                      </Grid>
+                    </Grid>
+                    <FieldArray name="$or">
+                      {({ push, remove }) => (
+                        <>
+                          {values.$or.map((parameters, index) => {
+                            return (
+                              <FieldGroup
+                                values={parameters}
+                                handleChange={handleChange}
+                                initialCustomSearchValues={
+                                  initialCustomSearchValues
+                                }
+                                fieldGroupIndex={index}
+                                removeFieldGroup={remove}
+                                key={index}
+                                setFieldValue={setFieldValue}
+                                push={push}
+                              />
+                            )
+                          })}
+
+                          <Grid
+                            item
+                            container
+                            direction="column"
+                            width="100%"
+                            alignItems={'center'}
+                            sx={{ mt: 1 }}
+                          >
+                            <Grid
+                              item
+                              container
+                              direction="row"
+                              width="756px"
```

Comment:
Can we use responsive values instead of a fixed width ? (maxWidth, minWidth; 'sm, 'md', 'lg')